src/components/Navbar/Navbar.jsx

```diff
@@ -1,13 +1,15 @@
 import { AuthModal } from "../../components";
 import { ASSETS_URL, ICONS_URL } from "../../constants";
-import { useAuth, useAuthModal } from "../../context";
+import { useAuth, useAuthModal, useFilter } from "../../context";
 import { toast } from "react-toastify";
-import { Link } from "react-router-dom";
+import { Link, useLocation, useNavigate } from "react-router-dom";
```

Comment:
`useNavigate` is not used in this file. If it's an unused import, you can delete it.

Reply:
Thanks for pointing it out! I've removed that now! I will push that in the next PR.